src/main.js

```diff
@@ -77,10 +87,38 @@ function displayHomeView() {
 }
 
 function displaySavedCovers() {
-  console.log("Is this working")
   savedCoversView.classList.remove("hidden");
   homeView.classList.add("hidden");
   showRandomButton.classList.add("hidden");
   saveCoverButton.classList.add("hidden");
   homeButton.classList.remove("hidden");  
-}
\ No newline at end of file
+}
+
+function displayUserCover() {
+  event.preventDefault();
+  getUserInput();
+  displayHomeView();
+}
+
+function pushUserInputToArray() {
+  covers.push(userCoverInput.value);
+  titles.push(userTitleInput.value);
+  descriptors.push(userDescriptionInput1 .value);
+  descriptors.push(userDescriptionInput2.value);
+}
+
+function getUserInput() {
+  var userBookCover = new Cover (
+    userCoverInput.value,
+    userTitleInput.value,
+    userDescriptionInput1.value,
+    userDescriptionInput2.value,
+  )
+  displayCover(userBookCover);
+}
+
+//declare a variable inside of a function and 
```

Comment:
We should clean up our pseudocode. So glad we are remembering to pseudocode and console log!